### DISCUSSION TITLE
How to change to the colorful icons in KDE Taskbar

### DISCUSSION THREAD

**Discussion by 0verEngineer** [SylEleuth/gruvbox-plus-icon-pack#98] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/b2fc4195-7531-4c1b-90d4-7b4580a76baf)

The icons in my KDE Taskbar are the Symbolic ones, in the Start Menu are the colored ones, can this be changed?

**Comment by SylEleuth** [comment, marked as answer]
There is a different folder for different size. If You look into `apps` subfolder You will see `16`, `22`, `24`, `32` and `48`. Depending the size used in DE, icons from that folder will be used. Anything below `32` links to `16`, above `32` to `scalable`. You can test it by increasing size of the task bar - at some point icons will change to colourful ones. If you don't want to use monochrome icons You would have to change `22` and `24` folders to link `scalable`, and change `16` into symlink.

**Reply by 0verEngineer** [reply]
Hey, thank you, it works with the change to the 22 and 24 symlinks, but i still have two questions:
- why is it like this?
- you say "and change 16 into symlink", where should the link point?

**Reply by SylEleuth** [reply]
> why is it like this?

If You want to use big 256px icon to display as 16px You might loose all details and not be able (in some cases) recognize what app is that. It might or not be same with slightly bigger icons but it all depends of Your screen resolution, HiDPI or eyesight. It has to have some defaults and You can change it as You like but would have to do that with every new release, or every time You make an new pull.

> you say "and change 16 into symlink", where should the link point?

To the folder with icons You want to display, `scalable` in Your case.

**Reply by 0verEngineer** [reply]
Ok thank you
